server/models/userModel.js

```diff
@@ -0,0 +1,26 @@
+const db = require("./db");
+
+module.exports = () => {
+  const createGenresTable = async () => {
+    try {
+      await db.query(`CREATE TABLE IF NOT EXISTS (
+      users 
+      Id SERIAL NOT NULL PRIMARY KEY,
```

Comment:
why is the id with capital letter, please change to  small letter and don't also forget to update where you are referencing it in the gif controller

Reply:
Ok sir